README.md

```diff
@@ -10,134 +10,68 @@ This is used by [Liftoff](http://github.com/tkellen/node-liftoff) to automatical
 
 ### extensions
 Map file types to modules which provide a [require.extensions] loader.
+
 ```js
 {
-  '.babel.js': 'babel/register',
+  '.babel.js': {
```

Comment:
should probably make this an array like node-jsx or make node-jsx a single object for consistency.  thoughts?